app/main/views/users.py

```diff
@@ -6,6 +6,9 @@
 from ... import data_api_client
 
 
+DEACTIVATED_USER_MESSAGE = "‘{user_name}’ ({user_email_address}) has been removed as a contributor."
```

Comment:
I don't think the old version had quotes around the `{user_name}` (however I'm not averse to them being put in!)

Reply:
Yeah I originally added quotes in most of the content strings that were missing them but then removed them as I didn't want the scope of this to creep, I noticed I missed this one as I was pushing and of course the tests don't catch it because previously most the time they just checked for the symbolic flash message.